sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StatsbeatTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.ConnectionString;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class StatsbeatTests
+    {
+        [Theory]
+        [InlineData("francecentral")]
+        [InlineData("francesouth")]
+        [InlineData("northeurope")]
+        [InlineData("norwayeast")]
+        [InlineData("norwaywest")]
+        [InlineData("swedencentral")]
+        [InlineData("switzerlandnorth")]
+        [InlineData("switzerlandwest")]
+        [InlineData("uksouth")]
+        [InlineData("ukwest")]
+        [InlineData("westeurope")]
+        public void StatsbeatConnectionStringIsSetBasedOnCustomersConnectionStringEndpointInEU(string euEndpoint)
+        {
+            var customer_ConnectionString = $"InstrumentationKey=1aa11111-bbbb-1ccc-8ddd-eeeeffff3333;IngestionEndpoint=https://{euEndpoint}.in.applicationinsights.azure.com/";
+            var parsedConectionString = ConnectionStringParser.GetValues(customer_ConnectionString);
+
+            // Set ConnectionString to null as it is static and may be impacted by other tests.
+            Statsbeat.s_statsBeat_ConnectionString = null;
```

Comment:
This is a red flag for me. 
We should be able to initialize and test the Statsbeat class without affecting other tests.

Reply:
Best practice is to use constructor and dispose to set `Statsbeat.s_statsBeat_ConnectionString` to null.

Reply:
by other tests it is referring to tests related to Statsbeat class itself. I have updated it to make it more clear. I will explore the dispose option in a separate follow up as that may touch the meterprovider part as well.